src/valuedlg.cpp

```diff
@@ -165,6 +165,7 @@ void ValueDlg::SetContent(QString id, QString key, QJsonValue value)
     m_queryPlan = "";
     if (value.isObject()) {
         if (m_key.startsWith("logical-query") ||
+            m_key.startsWith("aql-table") ||
```

Comment:
Do we have scenarios / plans where we will have keys that start with "aql-table" that aren't just "aql-table"? I think the reason we did startsWith for LQ is for things like "logical-query-rewrite".

Reply:
Emm, good question, but i don't know..Maybe we can do "startsWith", and then we don't need to modify it again when we have plans for "aql-table-xxx"